frontend/src/app/main/ui/workspace/sidebar/assets/colors.scss

```diff
@@ -68,20 +68,17 @@
 
 .name-block {
   @include bodyMedTipography;
-  display: grid;
-  grid-template-columns: auto 1fr;
   margin: 0;
   overflow: hidden;
-  .default-name-only,
-  .name {
-    color: var(--assets-item-name-foreground-color);
-    margin-right: $s-6;
-    @include textEllipsis;
-  }
-  .default-name {
-    min-width: 0;
-    color: var(--assets-item-name-foreground-color-rest);
-  }
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Q: why not using the mixin `textEllipsis` ?

Reply:
good question XD, I forgot